### PR TITLE
fix: remove smoke tests, eslinted, standardized repository @W-8222552@

### PR DESCRIPTION
**Pull request by WillieRuemmele**
### What does this PR do?
OSS salesforcedx repo
removes smoke tests

### CODE REVIEW

File: README.md
Comment (shetzel) [inline README.md:27]: I wonder if we want to make the example about installing latest-rc instead of pre-release since we're most likely moving away from pre-release?  @amphro

File: package.json
Comment (shetzel) [inline package.json:50]: Do we really need bluebird (and some of the others)?  I know it's just dev dependencies but it'd be nice to keep it as slim as possible, especially since smoke tests are being removed.

**Review by shetzel — COMMENTED** [review]
I don't see smoke tests being removed.  Or are they just not being copied over?

File: README.md
Comment (amphro) [reply README.md:27]: Ya, let's do that, and even explain the latest-rc schedule a little bit.

File: SETUP.md
Comment (amphro) [inline SETUP.md:11]: Let's remove this, and all references to this for now.

File: SETUP.md
Comment (amphro) [inline SETUP.md:9]: Change this to `main` and have that be the default branch in this repo.

File: SETUP.md
Comment (amphro) [inline SETUP.md:32]: This is internal information only, and no longer relevant. Please remove and make sure you **SQUASH** and merge this change to master before when you merge to main.

File: package.json
Comment (amphro) [reply package.json:50]: Agree, remove everything not needed.

File: package.json
Comment (amphro) [inline package.json:8]: Remove this, we do NOT want to ship shrinkwraps.

File: salesforcedx-49.13.1.sig
Comment (amphro) [inline salesforcedx-49.13.1.sig:1]: Delete this file. It should never have been checked into the internal repo to begin with.

File: README.md
Comment (amphro) [inline README.md:1]: Make sure you do NOT set up circleci for this repo yet. The current state of the jobs will not support releasing salesforcedx.

**Review by amphro — CHANGES_REQUESTED** [review]

File: README.md
Comment (amphro) [inline README.md:3]: Remove DX. It is just the Salesforce CLI.

Also, maybe say something like:

The Salesforce DX plugin aggregates all `force` topic commands into a single plugin that is bundled with the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli).

File: README.md
Comment (amphro) [inline README.md:26]: Remove this reference to pre-release

File: README.md
Comment (amphro) [inline README.md:12]: I think we can just remove this line and link to the CLI below.

File: README.md
Comment (amphro) [inline README.md:14]: i.e.

By default, the Salesforce DX plugin is bundled with the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli).

File: SETUP.md
Comment (amphro) [inline SETUP.md:1]: This setup seems way off. Just remove this whole thing.

File: README.md
Comment (amphro) [inline README.md:38]: Just remove this for now. But add a new section: 

## Issues

Please report any issues at https://github.com/forcedotcom/cli/issues

File: test/tsconfig.json
Comment (amphro) [inline test/tsconfig.json:1]: Why is this empty? Just remove the file then?

File: package.json
Comment (amphro) [inline package.json:27]: I'm not sure I love this name. Also, we only want the plugins to be autoupdated based on latest. I'm assuming that is what this is for?

**Review by amphro — CHANGES_REQUESTED** [review]

**Comment by amphro** [comment]
Can we remove the image too? Also, let's not forget to do a squash and merge when this is ready.

File: package.json
Comment (amphro) [inline package.json:28]: Let's remove apex. They aren't ready to be pinned.

**Review by amphro — APPROVED** [review]

**Review by shetzel — APPROVED** [review]

**Pull request merged**
